hw/ip/aes/lint/aes.vlt

```diff
@@ -8,4 +8,4 @@
 
 // Always_comb variable driven after use: 'regular'
 // regular is assigned in a for loop, regular[1] depends on regular[0]
-lint_off -msg ALWCOMBORDER -file "*/rtl/aes_key_expand.sv" -lines 206
+lint_off -msg ALWCOMBORDER -file "*/rtl/aes_key_expand.sv" -lines 208
```

Comment:
is there a way to specify this lint waiver using pattern matching instead of a line-based waiver?

lint_off [-rule <message>] [-file "<filename>"] [-match "<string>"]
https://www.veripool.org/ftp/verilator_doc.pdf

Reply:
Oh wow! I did not realize this got merged upstream! I will adapt the waiver accordingly.

Reply:
jep, pretty cool ;)...

Reply:
I have reverted to the `-lines` syntax to first switch to Verilator v4.028 including the framework we have in Ibex to warn when an older version is used. I will do the switch to the `-match` once we have that in.